pkg/upgrade/update.go

```diff
@@ -295,16 +295,9 @@ func updateInPlace(ctx context.Context, cs *api.OpenShiftManagedCluster, p api.P
 	return nil
 }
 
-func drain(
-	ctx context.Context,
-	cs *api.OpenShiftManagedCluster,
-	p api.Plugin,
-	vmc compute.VirtualMachineScaleSetVMsClient,
-	role api.AgentPoolProfileRole,
-	instanceID, nodeName string,
-) error {
+func (u *simpleUpgrader) delete(ctx context.Context, cs *api.OpenShiftManagedCluster, vmc compute.VirtualMachineScaleSetVMsClient, role api.AgentPoolProfileRole, instanceID, nodeName string) error {
```

Comment:
not sure the s/drain/delete/ makes it clearer

Reply:
delete is closer than drain to what this is actually doing :)